ios/brave-ios/Sources/AIChat/Subscription/SDK/BraveSkusSDK.swift

```diff
@@ -0,0 +1,346 @@
+// Copyright 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import Foundation
+import os.log
+import BraveCore
+
+// https://github.com/brave/brave-core/blob/master/components/skus/browser/rs/lib/src/models.rs#L137
+
+/// A structure representing the customer's credentials
+/// Returned by credentialsSummary
+public struct SkusCredentialSummary: Codable {
+  let order: SkusOrder
+  let remainingCredentialCount: UInt   // 512
+  let expiresAt: Date?                 // 2024-02-06T16:18:43
+  let active: Bool                     // true
+  let nextActiveAt: Date?              // 2024-02-06T16:18:43
+  
+  enum CodingKeys: String, CodingKey {
+    case order
+    case remainingCredentialCount = "remaining_credential_count"
+    case expiresAt
+    case active
+    case nextActiveAt
+  }
+}
+
+/// A structure representing the customer's order information
+/// Returned by refreshOrder
+public struct SkusOrder: Codable {
+  let id: String              // UUID
+  let createdAt: Date         // 2024-02-05T23:14:19.260973
+  let currency: String        // USD
+  let updatedAt: Date         // 2024-02-05T23:14:19.260973
+  let totalPrice: Double      // 15.0
+  let location: String        // leo.bravesoftware.com
+  let merchantId: String      // brave.com
+  let status: String          // paid
+  let expiresAt: Date?        // 2024-02-05T23:14:19.260973
+  let lastPaidAt: Date?       // 2024-02-05T23:14:19.260973
+  let items: [SkusOrderItem]
+  
+  enum CodingKeys: String, CodingKey {
+    case id
+    case createdAt
+    case currency
+    case updatedAt
+    case totalPrice
+    case location
+    case merchantId
+    case status
+    case items
+    // case metadata
+    case expiresAt
+    case lastPaidAt
+  }
```

Comment:
You don't need to manually define CodingKeys if you are not changing any of them